src/components/Navbar/Navbar.jsx

```diff
@@ -1,9 +1,10 @@
 import "./Navbar.css";
 import { Link } from "react-router-dom";
-import { useAuth } from "../../context";
+import { useAuth, useCart } from "../../context";
 
 const Navbar = () => {
   const { authState } = useAuth();
```

Comment:
just noticed that you have use `authState.token` couple of times in you code, you can just destructure it here in a single line for more convinience and cleaner code.
 `const { authState: {token} } = useAuth();`

Reply:
Done